config/application.rb

```diff
@@ -24,5 +24,9 @@ class Application < Rails::Application
     config.to_prepare do
       Doorkeeper::AuthorizationsController.layout 'application'
     end
+
+    config.action_cable.disable_request_forgery_protection = true
+    config.action_cable.url = '/api/cable'
```

Comment:
Just wanna make sure this is the URL we want to mount the cable at.

Reply:
No issues here